gatsby-config.js

```diff
@@ -61,6 +61,13 @@ const plugins = [
       path: `${__dirname}/src/content`,
     },
   },
+  {
+    resolve: 'gatsby-plugin-google-tagmanager',
+    options: {
+      id: '5LB3KK3',
```

Comment:
**RC:**  We should put it in a config file to keep it in one place (like other ids for external services)

Reply:
sure